src/domain/dtos/shlink.ts

```diff
@@ -0,0 +1,13 @@
+
+export class CreateShlinkDto {
+    userId: string
+    passcodeFailuresRemaining: number
```

Comment:
make this db or use case generated

Reply:
I think userId should come from the client side and passcodeFailuresRemaining is already defaulted to 5